helm_deploy/secrets-example.yaml

```diff
@@ -1,6 +1,8 @@
 ---
 secrets:
   APPINSIGHTS_INSTRUMENTATIONKEY: somesecretvalue
-  CLIENT_ID: somesecretvalue
-  CLIENT_SECRET: somesecretvalue
+  API_CLIENT_ID: clientid
+  API_CLIENT_SECRET: clientsecret
```

Comment:
sorry i'm getting ahead of myself here. in an upcoming PR i'm going to add seperate sets of creds for user facing login vs oauth2 secured api access. since this file is inconsequential in terms of deploy we can leave it for now if it's ok with you.